src/main/resources/metafile-schema.json

```diff
@@ -162,6 +169,21 @@
             "type": "string",
             "title": "ISO 4217 Currency CodeISO 42",
             "pattern": "^[A-Z]{3}$"
+          },
+          "payment_instrument_number": {
+            "id": "#/properties/payments/items/properties/payment_instrument_number",
+            "type": "string",
+            "title": "Cheque number in case of cheques ; postal order number in case of postal orders"
+          },
+          "sort_code": {
+            "id": "#/properties/payments/items/properties/sort_code",
```

Comment:
I think we will need validation such that if payment type is cheque then sort code should be present ?

Reply:
updated schema with validation and added unit tests